src/client-helpers.ts

```diff
@@ -200,7 +200,8 @@ export const cleanId = (path: string, method: string, suffix: string) => {
   return [method]
     .concat(path.split("/"))
     .concat(suffix)
-    .map((entry) => entry.replace(/[^A-Z0-9]/i, ""))
+    .map((entry) => entry.split(/[^A-Z0-9]/gi))
+    .flat(1)
```

Comment:
```suggestion
    .flat()
```

`1` [is the default](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flat#parameters)